plugins/inputs/logparser/README.md

```diff
@@ -89,8 +89,8 @@ current time.
   - drop     (drops the field completely)
 - Timestamp modifiers:
   - ts               (This will auto-learn the timestamp format)
-  - ts-ansic         ("Mon Jan _2 15:04:05 2006")
-  - ts-unix          ("Mon Jan _2 15:04:05 MST 2006")
+  - ts-ansic         ("Mon Jan \_2 15:04:05 2006")
+  - ts-unix          ("Mon Jan \_2 15:04:05 MST 2006")
```

Comment:
I know underscore is technically supposed to be escaped, but I prefer to leave them out for typeability and readability.  Github's markdown parser does handle it as desired without the escapes.  If you like we can use backtick's on these strings instead.

Reply:
Fair enough. I'll revert this change.